public/robots.txt

```diff
@@ -1,3 +1,6 @@
+User-agent: Twitterbot
+Disallow:
```

Comment:
Woudn't this let the bot crawl our catalog? This is not a good idea if so.

Reply:
it should allow the bot to crawl everything, so yes the catalog too. will change it allow `/downloads/` which is where the image is, and follow the rest of the rules. will that be enough?